### PR TITLE
Add Links to Employers and Resources in Career Results Modal

### PR DESCRIPTION
**Pull request by justinxu012**

### CODE REVIEW

File: src/components/paradox/results/personality/career/modal/employers.js [hunk 1]
```diff
@@ -18,7 +21,8 @@ CareerModalEmployers.propTypes = {
   employers: PropTypes.arrayOf(
     PropTypes.shape({
       name: PropTypes.string.isRequired,
-      description: PropTypes.string.isRequired
+      description: PropTypes.string.isRequired,
+      url: PropTypes.string.isRequired
```
Comment (tomprats) [inline src/components/paradox/results/personality/career/modal/employers.js:25]: I think the URL shouldn't be required (the code above works without it already)

File: src/components/paradox/results/personality/career/modal/resources.js [hunk 2]
```diff
@@ -18,7 +21,8 @@ CareerModalResources.propTypes = {
   resources: PropTypes.arrayOf(
     PropTypes.shape({
       name: PropTypes.string.isRequired,
-      description: PropTypes.string.isRequired
+      description: PropTypes.string.isRequired,
+      url: PropTypes.string.isRequired
```
Comment (tomprats) [inline src/components/paradox/results/personality/career/modal/resources.js:25]: I think the URL shouldn't be required (the code above works without it already)

File: src/components/paradox/results/personality/career/modal/employers.js (hunk 1 above)
Comment (justinxu012) [reply src/components/paradox/results/personality/career/modal/employers.js]: Good catch! Ok Removed!

File: src/components/paradox/results/personality/career/modal/resources.js (hunk 2 above)
Comment (justinxu012) [reply src/components/paradox/results/personality/career/modal/resources.js]: Great catch! Ok Removed!

**Review by tomprats — APPROVED** [review]

**Comment by tomprats** [comment]
Looks like the version jumped a little extra, this should just be 1.12.19

**Comment by justinxu012** [comment]
Added some styling changes requested by Josh.

**Review by tomprats — APPROVED** [review]

**Pull request merged**